### DISCUSSION TITLE
2023-5 - 泛函分析基础：学习指导和习题详解

### DISCUSSION THREAD

**Discussion by giscus** [duskmoon314/THU_BFA#10] [Announcements]
# 2023-5 - 泛函分析基础：学习指导和习题详解

https://duskmoon314.github.io/THU_BFA/ch6/2023-5.html

**Comment by zhengkw18** [comment]
第4问可以直接用内积空间最佳逼近元唯一（教材定理3.2.1）导出矛盾，然而鼠鼠忘了怎么证了，第3问也想不出来，宣布gg

**Reply by duskmoon314** [reply]
完全不记得还有这一条了，我也只想到内积空间有平行四边形等式
